frontend/src/sharedComponents/Metadata/ContentComps.jsx

```diff
@@ -198,6 +221,25 @@ const ContentExpanded = ({ metadata, showCategory, showMunicipality }) => {
         }
       }
     };
+    const getMetadataTypeName = async () => {
+      try {
+        const res = await fetch(`/api/MetadataType/${metadataTypeUuid}`);
+        if (res.status === 200) {
+          const { name } = await res.json();
+          setMetadataTypeName(name);
+        }
+      } catch (err) {
+        const { status } = err;
+        if (status === 404) {
+          dispatch(alertActions.error('Could not fetch metadata type name'));
```

Comment:
Could not find may be more specific in this case. Could not fetch indicates something else might have happened in the chain.
```suggestion
          dispatch(alertActions.error('Could not find metadata type name'));
```